src/docs/code-review.mdx

```diff
@@ -26,17 +36,13 @@ Additionally, language specific teams exist, primarily for SDKs:
 - [@getsentry/php](https://github.com/orgs/getsentry/teams/php)
 - [@getsentry/python](https://github.com/orgs/getsentry/teams/python)
```

Comment:
While we are here, we can probably get rid of the language specific teams (which are mostly now defunct). Instead I would prefer to link to https://open.sentry.io/structure/ which list all the GH teams as appropriate.